content/cli-v2.md

```diff
@@ -1546,6 +1546,26 @@ See [CPI config](cpi-config.md).
 
     See [Manual repair with Cloud Check](cck.md) for details.
 
+#### Create-Recovery-Plan {: #create-recovery-plan }
+
+- `bosh [GLOBAL-CLI-OPTIONS] create-recovery-plan PATH`
+
+    Checks for resource consistency and interactively prompts for problem resolution by instance group.  Optionally allows overriding the current value of `max_in_flight` for each instance group.  This information is then written to a YAML file and is used by the `bosh recover` command.
```

Comment:
How can I override the `max_in_flight` value  while generating the plan. I don't see any option for that.
Is this done manually in the generated plan afterwards? Then I would document that in the `recover` command only.

Reply:
I have added some examples - hope this clears it up!

Reply:
Ah, got it.